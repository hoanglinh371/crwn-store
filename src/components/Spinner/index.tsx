import React from 'react';
import classNames from 'classnames/bind';

import styles from './Spinner.module.scss';

const cx = classNames.bind(styles);

const Spinner: React.FC = () => {
  return (
    <div className={cx('overlay')}>
      <div className={cx('container')}></div>
    </div>
  );
};

export default Spinner;
