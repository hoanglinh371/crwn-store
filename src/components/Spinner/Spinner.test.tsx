import { render, cleanup } from '@testing-library/react';
import Spinner from '.';

describe('Spinner component', () => {
  afterEach(cleanup);

  it('Should_MatchSnapshot', () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
