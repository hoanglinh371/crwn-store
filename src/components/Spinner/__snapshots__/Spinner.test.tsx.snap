// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component Should_MatchSnapshot 1`] = `
<div>
  <div
    class="overlay"
  >
    <div
      class="container"
    />
  </div>
</div>
`;
